inspire_disambiguation/cli.py

```diff
@@ -113,15 +108,24 @@ def ethnicity_model(load_data_path, save_model_path):
     help=f"Number of processes to use. (default: '{conf['CLUSTERING_N_JOBS']}')",
     type=int,
 )
-def distance_model(ethnicity_model_path, save_model_path,
-                   sampled_pairs_size, test, train_fraction, n_jobs):
+def distance_model(
+    ethnicity_model_path,
+    save_model_path,
+    sampled_pairs_size,
+    test,
+    train_fraction,
+    n_jobs,
+):
     """Train distance model and save it."""
     click.secho("Starting Distance training.")
     test_signatures = train_and_save_distance_model(
-            ethnicity_model_path, save_model_path, sampled_pairs_size,
-            train_fraction)
+        ethnicity_model_path, save_model_path, sampled_pairs_size, train_fraction
+    )
     if test:
-        cluster(ethnicity_model_path, save_model_path, n_jobs, test_signatures)
+        click.secho("Starting clustering.")
+        cluster_from_redis(
```

Comment:
I don't think this is the right way to go. ```cluster_from_redis``` is just a method to start clustering on signature blocks received from redis. For example, by calling it in the testing, you will end up consuming things from the redis queue.

Reply:
So before we used cluster method for that. We can go for that, it should work (without calculating statistics per each block, we'll get one global statistic describing the classification of the whole dataset)